qcodes/instrument_drivers/rohde_schwarz/RTO1000.py

```diff
@@ -89,13 +89,13 @@ def get_raw(self):
                              'prepare_trace().')
 
         if instr.run_mode() == 'RUN Nx SINGLE':
-            N = instr.num_acquisitions()
-            M = instr.completed_acquisitions()
-            log.info('Acquiring {} traces.'.format(N))
-            while M < N:
-                log.info('Acquired {}:{} traces.'.format(M, N))
+            n = instr.num_acquisitions()
```

Comment:
well, perhaps better names are `total_acquisitions` for `n` and `completed_acquisitions` for `m` ? :) for readability